lib/Role/RoleForm/RoleForm.js

```diff
@@ -115,7 +116,23 @@ export const RoleForm = ({
                 />
               </Accordion>
 
+              <Pluggable
+                type="select-application"
+                checkedAppIdsMap={checkedAppIdsMap}
+                onSave={onSaveSelectedApplications}
+                renderTrigger={props => (
+                  <Button
+                    {...props}
+                    icon="plus-sign"
+                    disabled={isLoading}
+                  >
+                    <FormattedMessage id="stripes-authorization-components.crud.selectApplication" />
+                  </Button>
+                )}
+              />
```

Comment:
I think there should be a fallback in case a plugin of type `select-application` is not available:
```
<Pluggable ...>
  <FormattedMessage id="some-fallback-label" />
</Pluggable>
```